root/layout.tt

```diff
@@ -10,12 +10,22 @@
     <body>
         [%- React.embed(c, 'layout/components/Header') -%]
 
-        [%- IF c.user.is_editing_disabled -%]
+        [%- IF c.user.is_editing_disabled || c.user.are_notes_disabled -%]
             <div class="banner editing-disabled">
                 <p>
-                    [% l('You’re currently not allowed to edit or vote because an admin has revoked your privileges.
-                          If you haven’t already been contacted about why, please {uri|send us a message}.',
-                        { uri => { href => 'https://metabrainz.org/contact', target => '_blank' } }) %]
+                    [%- IF c.user.is_editing_disabled && c.user.are_notes_disabled -%]
+                        [% l('You’re currently not allowed to edit, vote or leave edit notes because an admin has revoked your privileges.
+                            If you haven’t already been contacted about why, please {uri|send us a message}.',
+                            { uri => { href => 'https://metabrainz.org/contact', target => '_blank' } }) %]
+                    [%- ELSIF c.user.is_editing_disabled -%]
+                        [% l('You’re currently not allowed to edit or vote because an admin has revoked your privileges.
+                            If you haven’t already been contacted about why, please {uri|send us a message}.',
+                            { uri => { href => 'https://metabrainz.org/contact', target => '_blank' } }) %]
+                    [%- ELSE -%]
+                        [% l('You’re currently not allowed to leave edit notes because an admin has revoked your privileges.
+                            If you haven’t already been contacted about why, please {uri|send us a message}.',
+                            { uri => { href => 'https://metabrainz.org/contact', target => '_blank' } }) %]
+                    [%- END -%]
```

Comment:
Since edits should be entered with edit notes, is there a case where editor would be allowed to enter edits without being allowed to enter edit notes?

Reply:
You can actually leave notes when entering the edit, but that's the only case where they can.

Reply:
Ok about editing, but you can still vote but not leave edit note, is it intended too?

Reply:
I'd say yes. This should be used on its own just for one use case IMO, getting people to calm down and not get into an argument. In most cases, it will be used in combination with editing + voting blocks, it's just that there *might* be some cases where @Freso only needs to block edit notes.